tests/test_rtdc_writer.py

```diff
@@ -350,11 +354,9 @@ def test_non_scalar_bad_shape():
     register_temporary_feature("peterpan", is_scalar=False)
     with RTDCWriter(exppath, mode="append") as hw:
         data = np.arange(10 * 3 * 5).reshape(10, 3, 5)
-        # This should work
-        hw.store_feature("peterpan", data, shape=(3, 5))
-        # This should not work
+        data = data.reshape(1, *data.shape)
```

Comment:
Maybe describe what you are doing here in a comment?

Reply:
Actually, this step is already covered in `write_image_grayscale()` method (see line number 370). So, I will remove this line from here. This line basically add an extra dimension served as an index in HDFview.

Reply:
It is fixed now. I removed the line 357 because it is not needed.